cPanel/cPanel_installonboot.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+## REQUIRED IN EVERY MARKETPLACE SUBMISSION
+# Add Logging to /var/log/stackscript.log for future troubleshooting
+exec > >(tee /dev/ttyS0 /var/log/stackscript.log) 2>&1
+
+# System Updates updates
+apt-get -o Acquire::ForceIPv4=true update -y
+DEBIAN_FRONTEND=noninteractive apt-get -y -o DPkg::options::="--force-confdef" -o DPkg::options::="--force-confold" install grub-pc
+apt-get -o Acquire::ForceIPv4=true update -y
+## END OF REQUIRED CODE FOR MARKETPLACE SUBMISSION
+
+set -e
```

Comment:
Postponed setting errexit option until after required apt-get code because script is designed to work other package managers.